app/command_utils.cc

```diff
@@ -0,0 +1,216 @@
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_CONSOLE),
+        ADD_UNTRANSLATED_COMMAND(TASK_MANAGER),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_DEVICES),
+        ADD_UNTRANSLATED_COMMAND(FEEDBACK),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BOOKMARK_BAR),
+        ADD_UNTRANSLATED_COMMAND(SHOW_HISTORY),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BOOKMARK_MANAGER),
+        ADD_UNTRANSLATED_COMMAND(SHOW_DOWNLOADS),
+        ADD_UNTRANSLATED_COMMAND(CLEAR_BROWSING_DATA),
+        ADD_UNTRANSLATED_COMMAND(IMPORT_SETTINGS),
+        ADD_UNTRANSLATED_COMMAND(OPTIONS),
+        ADD_UNTRANSLATED_COMMAND(EDIT_SEARCH_ENGINES),
+        ADD_UNTRANSLATED_COMMAND(VIEW_PASSWORDS),
+        ADD_UNTRANSLATED_COMMAND(ABOUT),
+        ADD_UNTRANSLATED_COMMAND(HELP_PAGE_VIA_KEYBOARD),
+        ADD_UNTRANSLATED_COMMAND(SHOW_APP_MENU),
+        ADD_UNTRANSLATED_COMMAND(MANAGE_EXTENSIONS),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_INSPECT),
+        ADD_UNTRANSLATED_COMMAND(BOOKMARKS_MENU),
+        ADD_UNTRANSLATED_COMMAND(SHOW_AVATAR_MENU),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_REQUEST_TABLET_SITE),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_TOGGLE),
+        ADD_UNTRANSLATED_COMMAND(TAKE_SCREENSHOT),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_FULLSCREEN_TOOLBAR),
+        ADD_UNTRANSLATED_COMMAND(INSTALL_PWA),
+        ADD_UNTRANSLATED_COMMAND(PASTE_AND_GO),
+        ADD_UNTRANSLATED_COMMAND(SHOW_FULL_URLS),
+        ADD_UNTRANSLATED_COMMAND(CARET_BROWSING_TOGGLE),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_QUICK_COMMANDS),
+
+        // Media
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_PLAYPAUSE),
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_MUTE),
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_LOOP),
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_CONTROLS),
+
+#if BUILDFLAG(ENABLE_SCREEN_AI_SERVICE)
+        // Screen AI Visual Annotations.
+        ADD_UNTRANSLATED_COMMAND(RUN_SCREEN_AI_VISUAL_ANNOTATIONS),
+#endif
+
+        // Tab search
+        ADD_UNTRANSLATED_COMMAND(TAB_SEARCH),
+        ADD_UNTRANSLATED_COMMAND(TAB_SEARCH_CLOSE),
+
+        // Brave Commands
+        ADD_UNTRANSLATED_COMMAND(SHOW_BRAVE_REWARDS),
+        ADD_UNTRANSLATED_COMMAND(NEW_TOR_CONNECTION_FOR_SITE),
+        ADD_UNTRANSLATED_COMMAND(NEW_OFFTHERECORD_WINDOW_TOR),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BRAVE_SYNC),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BRAVE_WALLET),
+        ADD_UNTRANSLATED_COMMAND(ADD_NEW_PROFILE),
+        ADD_UNTRANSLATED_COMMAND(OPEN_GUEST_PROFILE),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BRAVE_WALLET_PANEL),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BRAVE_VPN_PANEL),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_BRAVE_VPN_TOOLBAR_BUTTON),
+        ADD_UNTRANSLATED_COMMAND(MANAGE_BRAVE_VPN_PLAN),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_BRAVE_VPN),
+        ADD_UNTRANSLATED_COMMAND(COPY_CLEAN_LINK),
+        ADD_UNTRANSLATED_COMMAND(SIDEBAR_TOGGLE_POSITION),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_TAB_MUTE)
```

Comment:
Someday we might want to generate this with python, by parsing chrome_command_ids + brave_command_ids. But definitely not for this PR 😄

Reply:
Hahaha believe it or not, that's actually where I started. It worked pretty well but I had to go through manually and decide which ones needed to be exposed to this interface, as there are a lot of commands which only work when specific menus or submenus are opened. There's 15-30 commands about spell checking, which only work when you've right clicked on a misspell word :laughing: 

I figured if we have look at them manually anyway, my python script wasn't too useful :'( 

I was thinking of a more advanced version which only spits out the new commands since last time you updated, but wasn't sure how it should be enforced.

As you say though, something for a later PR :laughing: